gis/grass/Portfile

```diff
@@ -1,11 +1,300 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=2:ts=2:sts=2
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           wxWidgets 1.0
+PortGroup           github 1.0
+PortGroup           debug 1.0
 
+github.setup        OSGeo grass 8.0.0
 name                grass
-epoch               1
```

Comment:
The epoch can never be removed or decreased.

Reply:
fixed